internal/text/text_test.go

```diff
@@ -146,3 +146,33 @@ func TestFormatSlice(t *testing.T) {
 		})
 	}
 }
+
+func TestDisplayURL(t *testing.T) {
+	tests := []struct {
+		name string
+		url  string
+		want string
+	}{
+		{
+			name: "simple",
+			url:  "https://github.com/cli/cli/issues/9470",
+			want: "https://github.com/cli/cli/issues/9470",
+		},
+		{
+			name: "without scheme",
+			url:  "github.com/cli/cli/issues/9470",
+			want: "https://github.com/cli/cli/issues/9470",
+		},
+		{
+			name: "with query param and anchor",
+			url:  "https://github.com/cli/cli/issues/9470?q=is:issue#issue-command",
+			want: "https://github.com/cli/cli/issues/9470",
+		},
```

Comment:
Though I do not personally approve of it, I wouldn't be surprised if some GHES customers were hosting it insecurely.  As such, let's just make sure we're preserving the protocol:

```suggestion
		{
			name: "with query param and anchor",
			url:  "https://github.com/cli/cli/issues/9470?q=is:issue#issue-command",
			want: "https://github.com/cli/cli/issues/9470",
		},
		{
			name: "preserve http protocol use despite insecure",
			url:  "http://github.com/cli/cli/issues/9470",
			want: "http://github.com/cli/cli/issues/9470",
		},
```